components/molecules/AuthSection/auth-section.tsx

```diff
@@ -92,50 +96,56 @@ const AuthSection: React.FC = ({}) => {
     ],
   };
   return (
-    <div className="flex p-2 m-1 sm:py-0">
-      <div className="flex items-center gap-2 lg:gap-3">
-        {user ? (
-          <>
-            {onboarded === false ? (
-              <>
-                <OnboardingButton />
-                <Divider type="vertical" className="!h-6 !bg-light-slate-6"></Divider>
-              </>
-            ) : (
-              ""
-            )}
-            <button className="relative cursor-pointer" onClick={() => router.push(`/user/notifications`)}>
-              {userInfo && userInfo.notification_count > 0 && (
-                <span className="absolute right-0 block w-2 h-2 bg-orange-300 rounded-full"></span>
+    <>
+      <div className="flex p-2 m-1 sm:py-0">
+        <div className="flex items-center gap-2 lg:gap-3">
+          <SearchDialogTrigger />
+          {user ? (
+            <>
+              {onboarded === false ? (
+                <>
+                  <OnboardingButton />
+                  <Divider type="vertical" className="!h-6 !bg-light-slate-6"></Divider>
+                </>
+              ) : (
+                ""
```

Comment:
Refactor: `null` is prefered when not returning anything in JSX.

```suggestion
                null
```

or you can do `!onboarded && your_JSX`

Reply:
It was outside the scope of the PR at first!

But, I'll appropriate 🤞